applications/api.py

```diff
@@ -89,3 +102,34 @@ def get_required_submission_type(application):
         .values_list("application_step__submission_type", flat=True)
         .first()
     )
+
+
+def populate_interviews_in_jobma(application):
+    """
+    Go over each ApplicationStep for the application and create the interviews in Jobma.
+
+    Args:
+        application (BootcampApplication): A bootcamp application
+    """
+    for run_step in BootcampRunApplicationStep.objects.filter(
```

Comment:
We need to filter this to only include run steps with `application_step__submission_type=SUBMISSION_VIDEO`

Reply:
Thanks, good catch